ext/DecapodesCUDAExt.jl

```diff
@@ -109,6 +108,10 @@ function dec_cu_pair_wedge_product(::Type{Tuple{1,1}}, sd::HasDeltaSet2D)
     (α, β) -> dec_c_wedge_product(Tuple{1,1}, α, β, val_pack, Val{:CUDA}))
 end
 
+function dec_pair_wedge_product(::Type{Tuple{0,0}}, sd::HasDeltaSet)
```

Comment:
What is going in with this function?

Reply:
This to resolve ambiguities since we have a ```{0,k}``` and a ```{k,0}``` method but no ```{0,0}``` method, until now. This function is not actually called anywhere and shouldn't be called.

Reply:
It is also not exported so the user should never touch this function.